core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -765,7 +765,9 @@ class WorkspaceService(protected val userInfo: UserInfo,
               )
             }
           }
-          workspaces.map { workspace =>
+
+        workspaces.mapFilter { workspace =>
```

Comment:
The indenting has gotten off on this code (all the way down to line 809). Can you please fix that?